### PR TITLE
Abz- practicing first pr

### PR DESCRIPTION
**Pull request by ZaimAbz**

### CODE REVIEW

File: week-1/H-numbers/exercise.js [hunk 1]
```diff
@@ -1 +1,13 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var totalNum = numberOfStudents + numberOfMentors;
+var student ="Number of student :" + numberOfStudents;
+var mentor ="Number of mentors :" + numberOfMentors;
+var total ="Total number of students and mentors :" + totalNum;
```
Comment (foaly-nr1) [inline week-1/H-numbers/exercise.js]: Just a minor detail – please make sure to have a space before and after `=`. Consistency is important.

File: week-1/I-floats/exercise.js [hunk 2]
```diff
@@ -1,2 +1,12 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var total = numberOfStudents + numberOfMentors;
+var studentPerc = numberOfStudents * 100 / total;
+var mentorsPerc = numberOfMentors *100 / total;
+var roundStudentPerc = Math.round(studentPerc);
+var roundMentorsPerc = Math.round(mentorsPerc);
```
Comment (foaly-nr1) [inline week-1/I-floats/exercise.js]: Can you think of a way to use a function to make this more DRY (don't repeat yourself)?

File: week-1/J-functions/exercise2.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 function triple(number) {
+  return number * 2;
```
Comment (foaly-nr1) [inline week-1/J-functions/exercise2.js]: Double check the function name. Do the name and behaviour match?

File: week-1/H-numbers/exercise.js (hunk 1 above)
Comment (ZaimAbz) [reply week-1/H-numbers/exercise.js]: noted, thx

File: week-1/J-functions/exercise2.js (hunk 3 above)
Comment (ZaimAbz) [reply week-1/J-functions/exercise2.js]: lool sorry so true

File: week-1-practise/level-2/1-tax.js [hunk 4]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax() {
+  return price * 0.2;
+}
```
Comment (LucyMac) [inline week-1-practise/level-2/1-tax.js]: Hint: does your function need any parameters? ;)

File: week-1-practise/level-2/2-format-currency.js [hunk 5]
```diff
@@ -6,7 +6,9 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency() {
+  return "£" + (Math.round(price * 100) / 100).toFixed(2);
+}
```
Comment (LucyMac) [inline week-1-practise/level-2/2-format-currency.js]: Same comment as above

File: week-1-practise/level-2/3-currency-conversion.js [hunk 6]
```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD() {
+  return "$" + price * 1.4;
+}
```
Comment (LucyMac) [inline week-1-practise/level-2/3-currency-conversion.js]: And again as above.

Comment (LucyMac) [reply week-1-practise/level-2/3-currency-conversion.js]: Also, consider saving your exchange rate in a variable so we know what it is.

**Pull request closed without merging**